CONTRIBUTING.md

```diff
@@ -55,15 +55,16 @@ Roles are *voluntary* and reviewed periodically to ensure mutual clarity and car
 
 ### ⚡ Project Maintainer
 
-* **Qualification:** ***Black Belt*** in the program, completed a deep mindset exercise, and vetted by Organizational Ownership
+* **Qualification:** ***Black Belt*** in the program, completed [FEBI-4U](https://zenleader.global/programs/febi-4u), and vetted by Organizational Ownership
```

Comment:
How about ... fulfilled prerequisites for [FEBI Certification](https://zenleader.global/programs/febicertification)

Reply:
Actually ... completed the first step of the [Zen Leadership Instructor](https://zenleader.global/programs/coachinstructor/zlinstructor) pathway; and vetted by Organizational Ownership

The rationale is that the first step is FEBI-4U and this then syncs up nicely with the rest of the program.